sphinx_js/doclets.py

```diff
@@ -45,8 +48,17 @@ def gather_doclets(app):
                 d)
         except PathTaken as conflict:
             conflicts.append(conflict.segments)
+        except ParseError:
+            if not app.config.sphinx_js_lax:
+                raise
+            else:
+                logger.warning('Could not parse correctly %s' % d)
     if conflicts:
-        raise PathsTaken(conflicts)
+        if not app.config.sphinx_js_lax:
+            raise PathsTaken(conflicts)
+        else:
+            for conflict in conflicts:
+                logger.warning('Doclets conflict:%s' % conflict)
```

Comment:
It might be a little nicer to omit the "for" loop and instead print "Doclets conflict:" only once, then format ``conflicts`` afterward nicely.

Reply:
Done since I use PathsTaken now.